.github/workflows/scripts/release-prepare-release.sh

```diff
@@ -50,6 +50,11 @@ git add .
 git commit -m "make multimod-sync changes ${CANDIDATE_BETA}" || (echo "no multimod changes to commit")
 
 make gotidy
+
+pushd cmd/otelcontribcol
```

Comment:
The automated action failed because the go.sum for otelcontribcol was out of sync on the `make otelcontribcol` command. I added this extra `go mod tidy` here, as apparently the `make gotidy` explicitly rules out otelcontribcol. I see that this failed on the last release already, so I thought we should have a patch here so that we don't fail on the next release again, but we should definitely discuss what's the right way to fix this.